test/regression.uts

```diff
@@ -7038,8 +7038,6 @@ assert isinstance(pkt[LLTDAttributeEOP].payload, NoPayload)
 = Packet build / dissection
 pkt = Ether(raw(Ether(dst=ETHER_BROADCAST, src=RandMAC()) / LLTD(tos=0, function=0)))
 assert LLTD in pkt
-assert pkt.dst == pkt.real_dst
-assert pkt.src == pkt.real_src
```

Comment:
This makes literally no sense. I wonder how it worked previously :/
There's absolutely no method in `LLTD` that ensures that the `real_dst` would match the parent one

Reply:
I'm not sure to understand... I suppose this is used to test `LLTD.post_build()` code?

Reply:
You are absolutely right, I somehow missed the `post_build` function. Thanks !!